robottelo/ui/contentviews.py

```diff
@@ -271,6 +289,7 @@ def check_progress_bar_status(self, version):
         Checks the status of progress bar while publishing and
         promoting the CV to next environment
         """
+        timer = time.time() + 60 * 10
```

Comment:
Actually, I feel 60 * 10 is nice to have, It would quickly give us info, for how long the timeout value is set, without I needing to again do the division. I can make it "+ 600" if you think otherwise though. :)